src/main/java/de/tum/in/www1/artemis/service/feature/FeatureToggleService.java

```diff
@@ -46,9 +46,9 @@ public void init() {
 
         // Features that are neither enabled nor disabled should be enabled by default
         // This ensures that all features (except the Science API) are enabled once the system starts up
-        // Standardized Competencies are also disabled by default until the feature is ready
+        // Standardized Competencies and the Student Course Analytics Dashboard are also disabled by default until the feature is ready
         for (Feature feature : Feature.values()) {
```

Comment:
Could this for and the following ifs not be simplified to this?
```java
for (Feature feature : Feature.values()) {
    if (!features.containsKey(feature)) {
        features.put(feature, feature != Feature.Science && feature != Feature.StandardizedCompetencies && feature != Feature.StudentCourseAnalyticsDashboard);
    }
}
```
Alternatively you could also use a switch expression to make it even better.